### DISCUSSION TITLE
adding uninstall to the docs

### DISCUSSION THREAD

**Discussion by nicklm0** [JanDeDobbeleer/oh-my-posh#4132] [Troubleshoot]
Hi sir I'm interested contributing on the oh-my-posh project i would like to add  uninstall  on the oh-my-posh docs using winget for some folks who don't know how to uninstall

**Comment by JanDeDobbeleer** [comment]
@nickaldwin uninstalling isn't something that should be added to our docs imho, that's a package manager docs context.
